io-hdfs/src/main/java/cz/o2/proxima/storage/hdfs/HdfsStorage.java

```diff
@@ -32,7 +32,7 @@
 public class HdfsStorage extends StorageDescriptor {
 
   public HdfsStorage() {
-    super(Arrays.asList("hdfs"));
+    super(Arrays.asList("hdfs", "file"));
```

Comment:
I'd suggest the `hadoop:` prefix approach. Or we can move this outside this PR for now.

Reply:
Done